src/ansiblelint/schemas/main.py

```diff
@@ -104,6 +103,8 @@ def validate_file_schema(file: Lintable) -> list[str]:
                 message += "."
             message += f" See {documentation_url}"
         return [message]
+    else:
+        return [message]
     return []
```

Comment:
Additionally, this now highlights some weirdness that already existed. All of the `except`s already have returns, and now with the return in `else`, we can see that this `return []` will never get called.